### ISSUE TITLE
Media struct likely miss a string url

### ISSUE THREAD

**Issue by yuanshanhshan** [michimani/gotwi#247]
## In the resource ,field.go Media struct likely miss a string url
e.g. get a image from a tweet  by call twitter office api, should return a Media struct, and in the struct have a param url

**Comment by michimani** [comment]
@yuanshanhshan  
Thank you for reporting the issue!

Could you please provide me with some more information on this issue as I would like to resolve this issue. Specifically, could you please tell me which endpoint *"twitter office api"* represents?

**Comment by yuanshanhshan** [comment]
This is api url: [https://developer.twitter.com/en/docs/twitter-api/tweets/lookup/api-reference/get-tweets-id](url)
in the media fields have a "url"
![get-tweets-id](https://user-images.githubusercontent.com/84569701/170392532-562fbbf9-8494-4ce4-b268-0918c7bee9ad.png)

**Comment by yuanshanhshan** [comment]
My requirement is to get a image url from a tweet. And preview_image_url can not display in the HTML file.

**Comment by michimani** [comment]
Thank you very much.

This is an implementation omission to the `GetOutput` struct in `tweetlookup/types`.  
https://github.com/michimani/gotwi/blob/d562791aeae19aa8e5d4dea6ea78857fe85f0329/tweet/tweetlookup/types/response.go#L19-L21

Please wait a moment while we fix it.

**Comment by michimani** [comment]
@yuanshanhshan  
Thank you for your patience. We have released v0.11.4 which resolves this issue.

The sample code to get the media url of a tweet is as follows.

```go
p := &types.GetInput{
	ID: "1529841777561473026",
	Expansions: fields.ExpansionList{
		fields.ExpansionAttachmentsMediaKeys,
	},
	MediaFields: fields.MediaFieldList{
		fields.MediaFieldType,
		fields.MediaFieldUrl,
	},
}

t, err := tweetlookup.Get(context.Background(), c, p)
if err != nil {
	fmt.Println(err)
	return
}

fmt.Println(gotwi.StringValue(t.Data.Text))
fmt.Println(gotwi.StringValue(t.Includes.Media[0].URL))
```


Also, `preview_image_url` is only included in the response if the media is video. The sample code is as follows.

```go
p := &types.GetInput{
	ID: "1525817343385169920",
	Expansions: fields.ExpansionList{
		fields.ExpansionAttachmentsMediaKeys,
	},
	MediaFields: fields.MediaFieldList{
		fields.MediaFieldType,
		fields.MediaFieldPreviewImageUrl,
	},
}

t, err := tweetlookup.Get(context.Background(), c, p)
if err != nil {
	fmt.Println(err)
	return
}

fmt.Println(gotwi.StringValue(t.Data.Text))
fmt.Println(gotwi.StringValue(t.Includes.Media[0].PreviewImageUrl))
```

**Comment by yuanshanhshan** [comment]
Thank you for your help. I have solved the existing problems. Thank you again for your reply and help. @michimani
